### PR TITLE
Added BooleanServices.

### PR DESCRIPTION
**Pull request by ylussaud**

### CODE REVIEW

File: features/org.obeonetwork.m2doc.feature/codestyle/m2docCodeFormatter.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
 <setting id="org.eclipse.jdt.core.formatter.insert_space_after_opening_paren_in_annotation" value="do not insert"/>
 <setting id="org.eclipse.jdt.core.formatter.blank_lines_before_field" value="0"/>
 <setting id="org.eclipse.jdt.core.formatter.insert_space_after_opening_paren_in_while" value="do not insert"/>
-<setting id="org.eclipse.jdt.core.formatter.use_on_off_tags" value="false"/>
+<setting id="org.eclipse.jdt.core.formatter.use_on_off_tags" value="true"/>
 <setting id="org.eclipse.jdt.core.formatter.insert_space_between_empty_parens_in_annotation_type_member_declaration" value="do not insert"/>
 <setting id="org.eclipse.jdt.core.formatter.insert_new_line_before_else_in_if_statement" value="do not insert"/>
 <setting id="org.eclipse.jdt.core.formatter.insert_space_after_prefix_operator" value="do not insert"/>
```
Comment (nlepine) [inline features/org.obeonetwork.m2doc.feature/codestyle/m2docCodeFormatter.xml]: This file has nothing to do with the services, remove from the commit

File: plugins/org.obeonetwork.m2doc.sirius/.settings/org.eclipse.jdt.core.prefs [hunk 2]
```diff
@@ -284,7 +284,7 @@ org.eclipse.jdt.core.formatter.number_of_empty_lines_to_preserve=1
 org.eclipse.jdt.core.formatter.put_empty_statement_on_new_line=true
 org.eclipse.jdt.core.formatter.tabulation.char=space
 org.eclipse.jdt.core.formatter.tabulation.size=4
-org.eclipse.jdt.core.formatter.use_on_off_tags=false
+org.eclipse.jdt.core.formatter.use_on_off_tags=true
 org.eclipse.jdt.core.formatter.use_tabs_only_for_leading_indentations=false
 org.eclipse.jdt.core.formatter.wrap_before_binary_operator=true
 org.eclipse.jdt.core.formatter.wrap_before_or_operator_multicatch=true
```
Comment (nlepine) [inline plugins/org.obeonetwork.m2doc.sirius/.settings/org.eclipse.jdt.core.prefs]: This file has nothing to do with the services, remove from the commit

File: plugins/org.obeonetwork.m2doc.ui/.settings/org.eclipse.jdt.core.prefs [hunk 3]
```diff
@@ -284,7 +284,7 @@ org.eclipse.jdt.core.formatter.number_of_empty_lines_to_preserve=1
 org.eclipse.jdt.core.formatter.put_empty_statement_on_new_line=true
 org.eclipse.jdt.core.formatter.tabulation.char=space
 org.eclipse.jdt.core.formatter.tabulation.size=4
-org.eclipse.jdt.core.formatter.use_on_off_tags=false
+org.eclipse.jdt.core.formatter.use_on_off_tags=true
 org.eclipse.jdt.core.formatter.use_tabs_only_for_leading_indentations=false
 org.eclipse.jdt.core.formatter.wrap_before_binary_operator=true
 org.eclipse.jdt.core.formatter.wrap_before_or_operator_multicatch=true
```
Comment (nlepine) [inline plugins/org.obeonetwork.m2doc.ui/.settings/org.eclipse.jdt.core.prefs]: This file has nothing to do with the services, remove from the commit

File: plugins/org.obeonetwork.m2doc/.settings/org.eclipse.jdt.core.prefs [hunk 4]
```diff
@@ -284,7 +284,7 @@ org.eclipse.jdt.core.formatter.number_of_empty_lines_to_preserve=1
 org.eclipse.jdt.core.formatter.put_empty_statement_on_new_line=true
 org.eclipse.jdt.core.formatter.tabulation.char=space
 org.eclipse.jdt.core.formatter.tabulation.size=4
-org.eclipse.jdt.core.formatter.use_on_off_tags=false
+org.eclipse.jdt.core.formatter.use_on_off_tags=true
 org.eclipse.jdt.core.formatter.use_tabs_only_for_leading_indentations=false
 org.eclipse.jdt.core.formatter.wrap_before_binary_operator=true
 org.eclipse.jdt.core.formatter.wrap_before_or_operator_multicatch=true
```
Comment (nlepine) [inline plugins/org.obeonetwork.m2doc/.settings/org.eclipse.jdt.core.prefs]: This file has nothing to do with the services, remove from the commit

File: tests/org.obeonetwork.m2doc.sirius.tests/.settings/org.eclipse.jdt.core.prefs [hunk 5]
```diff
@@ -284,7 +284,7 @@ org.eclipse.jdt.core.formatter.number_of_empty_lines_to_preserve=1
 org.eclipse.jdt.core.formatter.put_empty_statement_on_new_line=true
 org.eclipse.jdt.core.formatter.tabulation.char=space
 org.eclipse.jdt.core.formatter.tabulation.size=4
-org.eclipse.jdt.core.formatter.use_on_off_tags=false
+org.eclipse.jdt.core.formatter.use_on_off_tags=true
 org.eclipse.jdt.core.formatter.use_tabs_only_for_leading_indentations=false
 org.eclipse.jdt.core.formatter.wrap_before_binary_operator=true
 org.eclipse.jdt.core.formatter.wrap_before_or_operator_multicatch=true
```
Comment (nlepine) [inline tests/org.obeonetwork.m2doc.sirius.tests/.settings/org.eclipse.jdt.core.prefs]: This file has nothing to do with the services, remove from the commit

File: tests/org.obeonetwork.m2doc.tests/.settings/org.eclipse.jdt.core.prefs [hunk 6]
```diff
@@ -284,7 +284,7 @@ org.eclipse.jdt.core.formatter.number_of_empty_lines_to_preserve=1
 org.eclipse.jdt.core.formatter.put_empty_statement_on_new_line=true
 org.eclipse.jdt.core.formatter.tabulation.char=space
 org.eclipse.jdt.core.formatter.tabulation.size=4
-org.eclipse.jdt.core.formatter.use_on_off_tags=false
+org.eclipse.jdt.core.formatter.use_on_off_tags=true
 org.eclipse.jdt.core.formatter.use_tabs_only_for_leading_indentations=false
 org.eclipse.jdt.core.formatter.wrap_before_binary_operator=true
 org.eclipse.jdt.core.formatter.wrap_before_or_operator_multicatch=true
```
Comment (nlepine) [inline tests/org.obeonetwork.m2doc.tests/.settings/org.eclipse.jdt.core.prefs]: This file has nothing to do with the services, remove from the commit

**Comment by ylussaud** [comment]
The formater tags are used in the BooleanServices class. If we do not commit this they will have no effect.

**Pull request merged**